docs/topics/extensions.rst

```diff
@@ -369,3 +369,13 @@ For more info see `Debugging in Python`_.
 This extension only works on POSIX-compliant platforms (i.e. not Windows).
 
 .. _Debugging in Python: https://pythonconquerstheuniverse.wordpress.com/2009/09/10/debugging-in-python/
+
+CheckSettings extension
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: CheckSettings
```

Comment:
The reason why other classes above are not fully qualified is that they are preceded by a `.. module::` declaration, which indicates that anything below, until the next `.. module::` declaration, belongs to the module.

As a result, this currently references `scrapy.extensions.debug.CheckSettings`.

You can either use the fully-qualified name, or add a `.. module::` declaration above in line with similar entries.